HCCGo/app/index.html

```diff
@@ -15,7 +15,7 @@
     <link href="lib/bootstrap/css/bootstrap.css" rel="stylesheet">
     <link href="lib/bootstrap-material-design/material.css" rel="stylesheet">
     <link href="lib/bootstrap-material-design/ripples.css" rel="stylesheet">
-    <link href="lib/c3/c3.css" rel="stylesheet">
+    <link href="../node_modules/c3/c3.css" rel="stylesheet">
```

Comment:
I'm worried, are these guaranteed to be available to the browser?  Seems odd to go up a directory.